lightningd/channel/channel_wire.csv

```diff
@@ -88,6 +88,7 @@ channel_accepted_htlc,0,forward,bool
 channel_accepted_htlc,0,amt_to_forward,u64
 channel_accepted_htlc,0,outgoing_cltv_value,u32
 channel_accepted_htlc,0,next_channel,struct short_channel_id
+channel_accepted_htlc,0,shared_secret,32
 
```

Comment:
I'm ok with these being identified as `sha256`, though it caught me a bit off-guard. I'll add a `secret` type to the generation script later.